Core/NoOpRunner.Core/Shapes/GeometricShape.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NoOpRunner.Core.Entities;
+
+namespace NoOpRunner.Core.Shapes
+{
+    public abstract class GeometricShape : BaseShape
+    {
+        public GeometricShape(int centerPosX, int centerPosY) : base(centerPosX, centerPosY)
+        {
+
+        }
+    }
+}
```

Comment:
I think we are expected to actually make use of the patterns, not just implement empty classes... The classes will have to remain and be used in the final codebase, we cannot just implement random shit and then throw it away.

Reply:
Changed abstract factory design implementation, but problem still exsists. What should I put in these classes?